### PR TITLE
[author][new] Added ConstraintJS Library@0.9.6

### PR DESCRIPTION
**Pull request by soney**
Origin of files: http://cjs.from.so/

### CODE REVIEW

File: ajax/libs/constraintjs/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "constraintjs",
+  "npmName": "constraintjs",
+  "npmFileMap": [
+    {
+      "basePath": "/dist/",
+      "files": [
+        "jquery.*"
```
Comment (PeterDaveHello) [inline ajax/libs/constraintjs/package.json]: I didn't see you commit `jquery.*`, maybe the config is wrong?

Comment (soney) [inline ajax/libs/constraintjs/package.json]: Thanks; removing

**Comment by PeterDaveHello** [comment]
Would you please try to enable auto-update on it? https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md

Maybe `repositories` and `authors` can be object not array, there is only one object there.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Pull request closed without merging**